src/tensorlake/documentai/models/enums.py

```diff
@@ -0,0 +1,121 @@
+"""
+Enums for document parsing.
+"""
+
+from enum import Enum
+
+
+class ChunkingStrategy(str, Enum):
+    """
+    Chunking strategy for parsing a document.
+
+    FRAGMENT: Each page element is converted into markdown form.
+    NONE: No chunking is applied.
+    PAGE: The document is chunked by page.
+    SECTION: The document is chunked into sections. Title and section headers are used as chunking markers.
+    """
+
+    FRAGMENT = "fragment"
+    NONE = "none"
+    PAGE = "page"
+    SECTION = "section"
+
+
+class FormDetectionMode(str, Enum):
+    """
+    Algorithm to use for detecting forms in a document.
+
+    VLM: Uses a VLM to identify questions and answers in a form.
+         Does not provide bounding boxes and is prone to hallucinations.
+    TSR: Uses a layout detector to identify questions and answers.
+         Does not work well with very complex forms.
+    """
+
+    VLM = "vlm"
+    TSR = "tsr"
```

Comment:
Are we sure `FormDetectionMode` is `VLM` or `TSR`? Isn't `TSR` for tables?

```suggestion
    ObjectDetection = "option_detection"
```

https://github.com/tensorlakeai/inkwell-server/blob/0431d13a52144de568aa7d011b199bcef906c747/server/src/api/v2/routes/parse/request_http_objects.rs#L568

Reply:
It used to be object detection before. TSR is closer to what happens internally, we retain the structure of the form.

Reply:
We changed it to TSR at the API level.